myia/abstract/utils.py

```diff
@@ -183,7 +182,13 @@ def pytype_to_abstract(main: tuple, args):
 @overload  # noqa: F811
 def pytype_to_abstract(main: list, args):
     arg, = args
-    return AbstractList(type_to_abstract(arg))
+    argt = type_to_abstract(arg)
+    assert argt is ANYTHING
```

Comment:
Why even ask for args if you're only ever going to use ANYTHING?

Reply:
Sanity check, to get a failure when trying to do `hastype(x, List[Int[64]])` and whatnot. It should be a better error, I suppose, but I thought I'd leave that for later.